src/test/java/seedu/address/logic/parser/CommandParserTestUtil.java

```diff
@@ -18,6 +18,9 @@ public static void assertParseSuccess(Parser<? extends Command> parser, String u
             Command expectedCommand) {
         try {
             Command command = parser.parse(userInput);
+            System.out.println("Actual: " + userInput);
+            System.out.println("Expected: " + expectedCommand);
```

Comment:
I think you forgot to remove these print statements

Reply:
thx sir